samples/Esprima.Benchmark/NodeListEnumerationBenchmark.cs

```diff
@@ -30,7 +28,7 @@ public void Setup()
     }
 
     [Benchmark]
-    public object For_DirectIndexing()
```

Comment:
Damn, nice oversight on my part... This pretty much messed up the benchmark results.

Nice catch! 👍